ase/scripts/generate_ase_environment.py

```diff
@@ -222,12 +222,9 @@ def config_sources(fd, filelist):
         if (len(s) == 0):
             None
         elif (s[0] == '+'):
-            # + simulator commands go in both Verilog and VHDL
+            # + simulator commands go only in Verilog.  VHDL simulation
+            # doesn't support these.
             vlog_srcs.append(s)
```

Comment:
Do we still support vhdl_srcs files?

Reply:
Yes, just not + directives since they don't work/aren't supported for VHDL.